CSS/index.css

```diff
@@ -143,9 +137,88 @@ section {
     width: 50%;
 }
 
-
 footer {
     text-align: center;
     text-decoration: underline;
     color: rgb(155, 143, 143);
-}
\ No newline at end of file
+}
+
+.grid {
+    display: grid;
+    grid-gap: 20px;
+    grid-template-rows: 2fr 4fr 2fr;
+    grid-template-columns: 2fr 4fr 2fr;
+    grid-template-areas:
+        "experience experience experience"
+        "connect connect connect";
+}
+
+.grid > * {
+    border: solid 1px blue;
+}
+
+@media (min-width: 640px) and (max-width: 789px) {
```

Comment:
**Suggestion:** Add comments before your media queries stating which devices they apply to (tablet, mobile, desktop). It will help spot the styles for specific device widths more easily.